swagger.yaml

```diff
@@ -867,69 +897,117 @@ definitions:
+          stateCode:
+            type: string
+            description: State code/abbreviation. This could also represent a province.
+            example: "OR"
+          postalCode:
+            type: string
+            example: "97331"
+          countyCode:
+            type: string
+            example: "41003"
+          nationCode:
+            type: string
   AddressesResultObject:
     properties:
       data:
         type: array
         items:
-          properties:
-            id:
-              type: string
-              description: Globally unique ID for the address.
-            type:
-              type: string
-              example: addresses
-            attributes:
-              properties:
-                addressType:
-                  type: string
-                  description: Code for the type of address record.
-                  example: CM
-                addressTypeDescription:
-                  type: string
-                  description: Short description of the address type.
-                  example: Current Mailing
-                addressLine1:
-                  type: string
-                  example: "123 Main Street"
-                addressLine2:
-                  type: string
-                  example: "Apartment 123"
-                addressLine3:
-                  type: string
-                addressLine4:
-                  type: string
-                houseNumber:
-                  type: string
-                city:
-                  type: string
-                stateCode:
-                  type: string
-                  description: State code/abbreviation. This could also represent a province.
-                  example: "OR"
-                state:
-                  type: string
-                  description: Human readable state code. This could also represent a province.
-                  example: "Oregon"
-                postalCode:
-                  type: string
-                  example: "97331"
-                countyCode:
-                  type: string
-                  example: "41003"
-                county:
-                  type: string
-                  description: Human readable county.
-                  example: "Benton"
-                nationCode:
-                  type: string
-                nation:
-                  type: string
-                  description: Human readable nation/country name.
-                lastModified:
-                  type: string
-                  format: date
-                  description: The date this address record was last modified.
+          $ref: "#/definitions/AddressResourceObject"
+  AddressResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/AddressResourceObject"
+  AddressResourceObject:
+    properties:
+      id:
+        type: string
+        description: Globally unique ID for the address.
+      type:
+        $ref: "#/definitions/AddressType"
+      attributes:
```

Comment:
I see a lot of the attributes here are also in `newAddressResourceObject`. I think it would be a good idea to define these separately and then `$ref` them.

Reply:
Good point. Fixed.